source/Octopus.Tentacle.Tests.Integration/ClientFileTransferRetriesTimeout.cs

```diff
@@ -22,14 +22,12 @@ namespace Octopus.Tentacle.Tests.Integration
     public class ClientFileTransferRetriesTimeout : IntegrationTest
     {
         [Test]
-        public async Task WhenRpcRetriesTimeOut_DuringUploadFile_TheRpcCallIsCancelled(
-            [Values]TentacleType tentacleType, 
-            [Values]bool stopPortForwarderAfterFirstCall, 
-            [Values]SyncOrAsyncHalibut syncOrAsyncHalibut)
+        [TentacleConfigurations(testStopPortForwarderAfterFirstCall: true)]
+        public async Task WhenRpcRetriesTimeOut_DuringUploadFile_TheRpcCallIsCancelled(TentacleConfigurationTestCase tentacleConfigurationTestCase)
         {
             PortForwarder portForwarder = null!;
-            await using var clientAndTentacle = await new ClientAndTentacleBuilder(tentacleType)
-                .WithAsyncHalibutFeature(syncOrAsyncHalibut.ToAsyncHalibutFeature())
+            await using var clientAndTentacle = await new ClientAndTentacleBuilder(tentacleConfigurationTestCase.TentacleType)
```

Comment:
I think it would have made sense to have `ClientAndTentacleBuilder` take a `tentacleConfigurationTestCase` or create an extension method which makes that builder from a `tentacleConfigurationTestCase`.

Since now when you add a new value to `tentacleConfigurationTestCase` you will have to go back to every test and make sure it is used.

Reply:
Yeah, that's a good idea. I'm gonna write it on my sharpening list 👍🏻

Reply:
This was actually pretty quick to do with a simple find/replace, so it's been done.